src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1736,6 +1736,12 @@ internal enum ErrorCode
 
         ERR_InternalError = 8751,
 
+        #region diagnostics introduced in preview
+
+        ERR_InvalidFunctionPointerCallingConvention = 8752,
```

Comment:
>8752 [](start = 54, length = 4)

Consider leaving some gap (for example starting with 8800) to simplify merges from other branches.

Reply:
I'm fine with dealing with the conflicts. Having a gap would annoy me :)

---
In reply to: [349347020](https://github.com/dotnet/roslyn/pull/39879#discussion_r349347020) [](ancestors = 349347020)

Reply:
> I'm fine with dealing with the conflicts. Having a gap would annoy me :)

As long as you are willing to adjust tests to reflect changes in numbers

---
In reply to: [349832744](https://github.com/dotnet/roslyn/pull/39879#discussion_r349832744) [](ancestors = 349832744,349347020)